docs/support.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Support"
+description: "Get help with Suga"
+---
+
+Need help with Suga? We're here to assist you.
+
+## Documentation
+
+Start with our comprehensive documentation:
+
+<CardGroup cols={2}>
+  <Card title="Quickstart" icon="rocket" href="/quickstart">
+    Get started in minutes
+  </Card>
+
+  <Card title="Core Concepts" icon="book" href="/core/overview">
+    Understand core concepts
+  </Card>
+
+  <Card title="Guides" icon="map" href="/guides">
+    Step-by-step tutorials
+  </Card>
+
+  <Card title="FAQ" icon="circle-question-mark" href="/faq">
```

Comment:
icon not loading

Reply:
Weird, I remember searching specifically for that one because the first one I tried didn't work. I'll check again